Archipelago.MultiClient.Net/Helpers/GameStateHelper.cs

```diff
@@ -0,0 +1,152 @@
+﻿using Archipelago.MultiClient.Net.Converters;
+using Archipelago.MultiClient.Net.Enums;
+using Archipelago.MultiClient.Net.Packets;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+
+namespace Archipelago.MultiClient.Net.Helpers
+{
+    public class GameStateHelper

```

Comment:
I think this should probably get a rename to something like "RoomStateHelper" or "MultiWorldSessionStateHelper". I feel as though "GameState" implies a lot that this isn't doing.

Reply:
I can see that, yes ill rename it